src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -57,10 +56,6 @@ public static FileResult analyzeFile(RepoConfiguration config, FileInfo fileInfo
         aggregateBlameAuthorInfo(config, fileInfo);
         fileInfo.setFileType(config.getFileType(fileInfo.getPath()));
 
-        if (config.isAnnotationOverwrite()) {
-            AnnotatorAnalyzer.aggregateAnnotationAuthorInfo(fileInfo, config.getAuthorEmailsAndAliasesMap());
```

Comment:
@niqiukun, thanks for the PR!
Just want to highlight that there is no need to remove the call to `aggregateAnnotationAuthorInfo` (line 61).
We still need to call `aggregateAnnotationAuthorInfo` for parsing any `@@author` tags provided by users (you may read about this feature [here](https://github.com/reposense/RepoSense/blob/master/docs/UserGuide.md#provide-data-using-author-tags))

The original issue was simply that the `RepoConfiguration` class no longer needs the `isAnnotationOverwrite` attribute.
It is sufficient to remove  the`isAnnotationOverwrite` attribute and the `isAnnotationOverwrite` / `setAnnotationOverwrite` methods.

Reply:
Thanks for the comment! Sorry I forgot that the default value for `isAnnotationOverwrite` is `true`. I have fixed this.